Packs/MalwareInvestigationAndResponse/pack_metadata.json

```diff
@@ -11,8 +11,13 @@
     "categories": [
         "Endpoint"
     ],
-    "tags": [],
-    "useCases": [],
+    "tags": [
+        "Malware",
+        "Use Case"
+    ],
+    "useCases": [
+        "Malware"
+    ],
```

Comment:
for use cases - please add the following "Hunting", "Incident Response"

Reply:
Done.